src/rdata.cpp

```diff
@@ -188,11 +190,11 @@ void ReturnData::processPreEquilibration(SteadystateProblem const &preeq,
         for (int ip = 0; ip < nplist; ip++)
             writeSlice(sx_rdata[ip], slice(sx_ss, ip, nx));
     }
-    /* Get cpu time for Newton solve in seconds */
-    preeq_cpu_time = preeq.getCPUTime() / 1000;
-    preeq_status = static_cast<int>(preeq.getNewtonStatus());
+    /* Get cpu time for Newton solve in milliseconds */
```

Comment:
cpu time for simulation is reported in ms though, right?

Reply:
I wanted to have things in the same unit... Now, everything should be in milliseconds, hopefully, but will recheck